programs/sbf/tests/programs.rs

```diff
@@ -1407,7 +1405,7 @@ fn assert_instruction_count() {
             (program_key, AccountSharedData::new(0, 0, &loader_id)),
             (
                 Pubkey::new_unique(),
-                AccountSharedData::new(0, 8, &program_key),
+                AccountSharedData::new(0, 0, &program_key),
```

Comment:
Was 8 an incorrect value, or is it a cleanup?

Reply:
A `post_adjustments` closure was added to `mock_process_instruction()`.
Thus, the value of the expected CU consumption can now be transported by the scope,
and does not need an instruction account anymore.